### PR TITLE
Use `help-all.json` to label prereleases in version selector

### PR DESCRIPTION
**Pull request by tgolsson**
This ensures that when we release a version, it'll not tag it as a prerelease.

### CODE REVIEW

**Comment by thejcannon** [comment]
I suspect the only way to do this correctly would be to load the help-all JSON and look at some version key

**Comment by tgolsson** [comment]
@thejcannon Agreed, but let's triage it like this and fix it properly in a later PR?

**Comment by thejcannon** [comment]
Yeah, sorry I meant to point out (but it slipped my mind) is that when we create 2.21.dev0, I suspect 2.20.0a0 will meet this criteria so it won't be marked as prerelease.

So this patch buys us a week or two, but left alone might cause some serious confusion.

If doing the right thing is like 20% harder, I think we ought to try up front to avoid that confusion.

Otherwise, if it isn't, I'll +1

**Comment by tgolsson** [comment]
I can maybe look tomorrow at a better solution; since I know that the data exists now.

**Comment by thejcannon** [comment]
That would be amazing ♥️

**Comment by tgolsson** [comment]
@thejcannon PTAL! I'm not sure where I could write a test for this, but I did a check locally and it seems to work as expected.

File: docusaurus.config.js [hunk 1]
```diff
@@ -47,6 +50,30 @@ const formatCopyright = () => {
...
+  const pantsVersion = helpAll["scope_to_help_info"][""]["advanced"].find(
+    (help) => help["config_key"] === "pants_version"
+  );
+
+  const hardcoded = pantsVersion["value_history"]["ranked_values"].find(
+    (value) => value["rank"] == "HARDCODED"
+  );
+
+  // Check if it's one of xx.xx.xx.dev0, xx.xx.xxa0, xx.xx.xxrc0
+  const rex = /^(\d+\.\d+\.\d+)(\.dev|a|rc)\d+$/;
```
Comment (thejcannon) [inline docusaurus.config.js:72]: What about beta?

**Review by thejcannon — COMMENTED** [review]
You'll want to update the PR title and description too!

Great work!

File: docusaurus.config.js (hunk 1 above)
Comment (tgolsson) [reply docusaurus.config.js:72]: I don't think I've ever seen a `b` suffix on any Pants release, but can add it.

Comment (thejcannon) [reply docusaurus.config.js]: The fewer assumptions the better IMO. Thanks!

**Review by thejcannon — APPROVED** [review]

**Pull request merged**